app/package.json

```diff
@@ -19,7 +19,7 @@
     "@react-navigation/native": "^5.8.10",
     "@react-navigation/stack": "^5.12.8",
     "axios": "^0.21.1",
-    "expo": "^40.0.0",
+    "expo": "^41.0.0",
```

Comment:
is there a reason why you updated this? This might have breaking changes in other areas of the app.

Reply:
I think this is ok, and shows initiative on keeping things up to date.